From 92ed618c4826e1eacf6740d0c7e2cb7372bb235f Mon Sep 17 00:00:00 2001
From: Petr Lautrbach <plautrba@redhat.com>
Date: Wed, 9 Mar 2016 17:14:25 +0100
Subject: [PATCH 1/2] Remove src/run_cmd.py from POTFILES

---
 po/POTFILES.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/po/POTFILES.in b/po/POTFILES.in
index 44633f5..2699f12 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -15,7 +15,6 @@ src/log.py
 src/Plugin.py
 src/rpc.py
 src/rpc_interfaces.py
-src/run_cmd.py
 src/server.py
 src/serverconnection.py
 src/signature.py
-- 
2.5.0

